language/stdlib/modules/libra_system.move

```diff
@@ -335,14 +335,20 @@ module LibraSystem {
    }
 
    fun reconfigure_() acquires ValidatorSet {
+       // Do not do anything if time is not set up yet.
+       if(LibraTimestamp::is_genesis()) {
+           return ()
```

Comment:
Should/can this abort rather than returning silently?

Reply:
Returning silently seems fine. This is just to help the genesis generation process.